vue.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     proxy: { "/api": { target: process.env.VUE_APP_SERVER_URL} }
   },
   configureWebpack: {
+    devtool: 'source-map',
```

Comment:
This is a drive-by improvement from [the docs](https://v2.vuejs.org/v2/cookbook/debugging-in-vscode.html#Displaying-Source-Code-in-the-Browser), which I used to make debugging easier.